partner-portal/src/UI/src/app/search/search.component.scss

```diff
@@ -0,0 +1,96 @@
+mat-card {
+    margin-bottom: 24px;
+  }
+  
+  .mat-mdc-card {
+    border: 1px solid gray;
+    box-shadow: none !important;
+    margin: 30px 0 30px 0;
+    padding: 12px;
+  
+    .mat-mdc-card-header {
+      padding-top: 12px;
+  
+      .mat-mdc-card-header-text {
+        width: 100%;
+        vertical-align: middle;
+      }
+    }
+  }
+
+  .form-label {
+    font-family: "BCSans-Bold", "BC Sans Bold", "BC Sans", sans-serif;
+    font-weight: 700;
+    font-style: normal;
+    font-size: 16px;
+    line-height: 10px;;
+  }
+  
+  .search{
+    display: flex; 
+    flex-direction: row; 
+    gap: 32px;
+    padding-top: 32px;
+  }
+  .search-content{
+    display: flex;
+    flex-direction: row;
+  }
+
+  .driver{
```

Comment:
I'm worried we are gonna have leftover CSS from temporary code

Reply:
This is a new page and these are the new styles added for this page not related to any previous pages on other portals.